core/modules/system/system.install

```diff
@@ -27,13 +27,22 @@ function system_requirements($phase) {
     $profile = backdrop_get_profile();
     if ($profile != 'standard') {
       $info = system_get_info('module', $profile);
-      $requirements['install_profile'] = array(
-        'title' => $t('Install profile'),
-        'value' => $t('%profile_name (%profile-%version)', array(
+      if (!empty($info['version'])) {
+        $value = $t('%profile_name (%profile-%version)', array(
```

Comment:
Simpler? 🙂

```php
      // If a version has been specified, append it to the profile.
      $profile .= !empty($info['version']) ? '-' . $info['version'] : '';
      $requirements['install_profile'] = array(
        'title' => $t('Install profile'),
        'value' => $t('%profile_name (%profile)', array('%profile_name' => $info['name'], '%profile' => $profile)),
        'severity' => REQUIREMENT_INFO,
        'weight' => -9
      );
```

Reply:
You're too clever for me :stuck_out_tongue: 
PR updated.